### ISSUE TITLE
[bug] Markdown generator 'cmake' instructions not working

### ISSUE THREAD

**Issue by aleksa2808** [conan-io/conan#6773]
Hi,
The new Markdown generator idea seems really cool! However, when I follow the instructions for linking a dependency using the `cmake` generator, CMake says it can't find the dependency target.   
Using boost/1.70.0 as an example, the markdown file says to link against the `boost::boost` target, but shouldn't it be `CONAN_PKG::boost`?

### Environment Details (include every applicable attribute)
  * Operating System+version: Ubuntu 18.04
  * Compiler+version: Clang 9
  * Conan version: 1.24.0
  * Python version: 3.6.9

### Steps to reproduce (Include if Applicable)
Configure a project with a `boost/1.70.0` dependency, and `cmake` and `markdown` generators. Then, just follow the instructions in the generated markdown file for the `cmake` generator.

#### conanfile.txt
```
[requires]
boost/1.70.0

[generators]
cmake
markdown
```

#### CMakeLists.txt
```
cmake_minimum_required(VERSION 3.10)

include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup(TARGETS)

add_executable(main main.cpp)
target_link_libraries(main boost::boost)
```

#### main.cpp
```
int main(){}
```

### Logs (Executed commands with output) (Include/Attach if Applicable)

```
$ mkdir build && cd build
$ conan install ..
$ cmake .. -DCMAKE_C_COMPILER=clang-9 -DCMAKE_CXX_COMPILER=clang++-9
-- The C compiler identification is Clang 9.0.0
-- The CXX compiler identification is Clang 9.0.0
-- Check for working C compiler: /usr/bin/clang-9
-- Check for working C compiler: /usr/bin/clang-9 -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/clang++-9
-- Check for working CXX compiler: /usr/bin/clang++-9 -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: Adjusting output directories
-- Conan: Using cmake targets configuration

... (conan dep found lines)

-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Current conanbuildinfo.cmake directory: /path/to/dir
-- Conan: Compiler Clang>=8, checking major version 9
-- Conan: Checking correct version: 9
-- Configuring done
CMake Error at CMakeLists.txt:6 (add_executable):
  Target "main" links to target "boost::boost" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


-- Generating done
-- Build files have been written to: /path/to/dir
```

**Comment by memsharded** [comment]
Hi @aleksa2808 

Yes, good catch. The boost::boost target is for the ``cmake_find_package`` generator, not for the ``cmake`` one. Lets fix it for the next release. Thanks for reporting!

**Comment by aleksa2808** [comment]
Hey, since it's a really minor issue, I already submitted a fix for it, hope you don't mind! :smile:

**Comment by memsharded** [comment]
Of course not, thanks very much for contributing it! 😄

**Comment by czoido** [comment]
closed by #6788
